security/fma-incident-response-improvements.md

```diff
@@ -0,0 +1,246 @@
+    - [Description](#description-2)
+    - [Risk Assessment](#risk-assessment-2)
+      - [Mitigations](#mitigations-2)
+      - [Detection](#detection-2)
+      - [Recovery Path(s)](#recovery-paths-2)
+      - [Action items:](#action-items-1)
+  - [FM4: Errors in bond distribution](#fm4-errors-in-bond-distribution)
+    - [Description](#description-3)
+    - [Risk Assessment](#risk-assessment-3)
+    - [Mitigations](#mitigations-3)
+    - [Detection](#detection-3)
+    - [Recovery Path(s)](#recovery-paths-3)
+    - [Action items](#action-items-2)
+    - [Generic items we need to take into account:](#generic-items-we-need-to-take-into-account)
+  - [Action Items](#action-items-3)
+  - [Audit Requirements](#audit-requirements)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+_Italics are used to indicate things that need to be replaced._
+
+|                    |                |
+| ------------------ | -------------- |
+| Author             | wildmolasses   |
+| Created at         | 2025-01-22     |
+| Initial Reviewers  | smartcontracts |
+| Need Approval From |                |
+| Status             | Draft          |
+
+## Introduction
+
+This document covers the fault dispute game incident response improvements project. The project makes modifications to several key contracts to improve incident response capabilities:
+
+1. DelayedWETH: Simplified hold functionality for emergency response
+2. OptimismPortal: Improved game type management and withdrawal validation
+3. AnchorStateRegistry: Unified anchor state and improved validation
+4. FaultDisputeGame: Added bond refunding for invalidated games
+
+Below are references for this project:
+
+- Design Docs
+  - [Anchor State Poisoning Protection](https://github.com/ethereum-optimism/design-docs/blob/b807dc5880f48a3631222e7f519c02a7668f2493/protocol/proofs/anchor-state-poison-protection.md)
+  - [DelayedWETH Expansion](https://github.com/ethereum-optimism/design-docs/blob/b807dc5880f48a3631222e7f519c02a7668f2493/protocol/proofs/delayed-weth-expansion.md)
+  - [FDG Refund Mode](https://github.com/ethereum-optimism/design-docs/blob/b807dc5880f48a3631222e7f519c02a7668f2493/protocol/proofs/fdg-refund-mode.md)
+  - [Withdrawal Invalidation Mitigation](https://github.com/ethereum-optimism/design-docs/blob/b807dc5880f48a3631222e7f519c02a7668f2493/protocol/proofs/withdrawal-invalidation-mitigation.md)
+- Specifications
+  - [Fault Dispute Game/Bond Incentives - DRAFT](https://github.com/ethereum-optimism/specs/pull/524/files)
+  - [Portal - DRAFT](https://github.com/ethereum-optimism/specs/pull/519/files)
+  - [AnchorStateRegistry - DRAFT](https://github.com/ethereum-optimism/specs/pull/517/files)
+  - [DelayedWETH - DRAFT](https://github.com/ethereum-optimism/specs/blob/a0c94920a3c3b6b3527e4f9c3f07b787cd9fe2ad/specs/fault-proof/stage-one/bond-incentives.md#delayedweth)
```

Comment:
Why are the specs labeled as DRAFT, does this indicate something is not finalized? Also the "Fault Dispute Game/Bond Incentives" one links to a closed PR that was not merged, so we should update that link

Reply:
Resolved in 855cf68